docker/docker-compose.yml

```diff
@@ -160,9 +177,19 @@ services:
       - MONGO_HOST=mongo
     volumes: *default-scripts
     # TODO: Remove this after startup improvement fixes
-    entrypoint: ["wait-for-it.sh", "-h", "hypertrace-federated-service", "-p", "9002", "--timeout=120",
-                 "--" ,
-                 "start-bootstrapper.sh"]
+    entrypoint:
+      [
+        "wait-for-it.sh",
+        "-h",
+        "hypertrace-federated-service",
+        "-p",
+        "9002",
+        "--timeout=120",
+        "--",
+        "start-bootstrapper.sh",
+      ]
+    healthcheck:
+      start_period: 180s
```

Comment:
Is this needed?

Reply:
Copied from your PR I think. I could remove it easily.